test/cri-containerd/container_virtual_device_test.go

```diff
@@ -467,4 +512,241 @@ func Test_RunContainer_VirtualDevice_ClassGUID_WCOW_Process(t *testing.T) {
 
 	containerID := createContainer(t, client, ctx, containerRequest)
 	runContainerLifetime(t, client, ctx, containerID)
+
+	verifyGPUIsPresentWCOW(t, client, ctx, containerID)
+}
+
+func Test_RunContainer_VirtualDevice_GPU_WCOW_Hypervisor(t *testing.T) {
+	requireFeatures(t, featureWCOWHypervisor, featureGPU)
+
+	if osversion.Get().Build < 19566 {
+		t.Skip("Requires build +19566")
+	}
+
+	testDeviceInstanceID, err := findTestNvidiaGPUDevice()
+	if err != nil {
+		t.Skipf("skipping test, failed to retrieve assignable device on host with: %v", err)
```

Comment:
Now that the user has to explicitly specify they want the GPU feature tested, we should probably fail instead of skipping if we don't find a GPU.